autoload/neosnippet/view.vim

```diff
@@ -247,6 +247,13 @@ function! neosnippet#view#_search_snippet_range(start, end, cnt, ...) "{{{
     return 1
   endfor
 
+  for linenum in range(a:start, a:end)
+    let tmp_line = getline(linenum)
+    let tmp_line = substitute(tmp_line, '\v\%uc\(([^)]+)\)', '\U\1\E', 'g')
```

Comment:
Bro, you can modify and pass by my pull request, it's no matter.Cause actually I not find good way else to do it.....